internal/interceptor/server_test.go

```diff
@@ -521,7 +515,23 @@ func TestWebhookHandling(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			res, err := http.Post(ts.URL, "application/json", f)
+			body, err := ioutil.ReadAll(f)
+			if err != nil {
+				t.Fatal(err)
+			}
+			fr := bytes.NewReader(body)
+			req, err := http.NewRequest("POST", ts.URL, fr)
+			if err != nil {
+				t.Fatalf("NewRequest: %v", err)
+			}
+			if tc.wrongSignature {
+				req.Header.Set(signatureHeader, "foobar")
+			} else {
+				req.Header.Set(signatureHeader, hmacHeader(t, testWebhookSecret, body))
+			}
+			req.Header.Set("Content-Type", "application/json")
+			client := &http.Client{}
```

Comment:
Would it make sense to set a timeout here?

Reply:
Added